responsibleai_text/responsibleai_text/rai_text_insights/rai_text_insights.py

```diff
@@ -269,7 +280,9 @@ def _validate_model(self, model: Any, test: pd.DataFrame,
             target_column, axis=1)
         small_test_data = get_text_columns(small_test_data, text_column)
         small_test_data = small_test_data.iloc[0]
-        if task_type != ModelTask.QUESTION_ANSWERING:
+        if task_type not in [
```

Comment:
nit: I think this might look a bit cleaner
```
list_task_outputs = [ModelTask.QUESTION_ANSWERING,
                                ModelTask.GENERATIVE_TEXT]
if task_type not in list_task_outputs:
  ...
```